pcm-player.js

```diff
@@ -143,23 +147,30 @@ PCMPlayer.prototype.flush = function() {
             offset += this.option.channels;
         }
     }
-    
+
     if (this.startTime < this.audioCtx.currentTime) {
         this.startTime = this.audioCtx.currentTime;
     }
     bufferSource.buffer = audioBuffer;
     bufferSource.connect(this.gainNode);
     bufferSource.start(this.startTime);
+    const feedCounter = this.feedCounter;
+    bufferSource.onended = () => {
```

Comment:
what javascript version are we targeting in this file?

Reply:
That's a good question. @megamk

Reply:
one that supports arrow functions and used to build sdk with webpack, so it's safe. I'm building using 8.9.4